packages/terra-form-select/src/MultiSelect.jsx

```diff
@@ -214,7 +206,7 @@ class MultiSelect extends React.Component {
         display={this.display()}
         onDeselect={this.handleDeselect}
         onSelect={this.handleSelect}
-        placeholder={selectPlaceholder}
+        placeholder={intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' })}
```

Comment:
Do we need `placeholder={intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' })}` prop assignment here ...?? because I see input of multi-select always refers to ['Terra.form.select.defaultDisplay'](https://github.com/cerner/terra-core/blob/cf1f413a32cc4702892ba571ceae14c7e0a45404/packages/terra-form-select/src/multiple/Frame.jsx#L253) and `placeholder` prop has no reference in `frame.jsx`. and we can see [placeholder prop is being set to undefined as well](https://github.com/cerner/terra-core/blob/cf1f413a32cc4702892ba571ceae14c7e0a45404/packages/terra-form-select/src/multiple/Frame.jsx#L710).

Reply:
Oops!! forgot to remove it. Done here https://github.com/cerner/terra-core/pull/3094/commits/da8765dc02ec816efca769497d1f26ba9503f98d